<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>247</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Meu Dragãozinho</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/imgs/imgs/dragon1.png</normaloff>:/imgs/imgs/dragon1.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Cidade:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QComboBox" name="cityList"/>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QTableWidget" name="itemsTable">
      <property name="selectionMode">
       <enum>QAbstractItemView::MultiSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>140</number>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>100</number>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>100</number>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>100</number>
      </attribute>
      <column>
       <property name="text">
        <string>Item</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Gastos (R$)              </string>
       </property>
      </column>
      <column>
       <property name="text">
        <string> Peso                           </string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Inflação(mensal)</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Inflação(12 meses)</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="addItem">
      <property name="text">
       <string>Adicionar Item</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="removeItem">
      <property name="text">
       <string>Remover Item</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" rowspan="2">
     <widget class="QLabel" name="message">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="QLabel" name="label_2">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>100</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/imgs/imgs/dragon1.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>addItem</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>openAddItemDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>476</x>
     <y>453</y>
    </hint>
    <hint type="destinationlabel">
     <x>471</x>
     <y>453</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeItem</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>removeItems()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>575</x>
     <y>463</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>459</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cityList</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateInflation(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>395</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>547</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openAddItemDialog()</slot>
  <slot>calculateInflation()</slot>
  <slot>removeItems()</slot>
  <slot>updateInflation(QString)</slot>
 </slots>
</ui>
